packages/web-components/src/components/va-textarea/va-textarea.tsx

```diff
@@ -126,19 +126,22 @@ export class VaTextarea {
   render() {
     const { label, error, placeholder, name, required, value } = this;
     const maxlength = this.getMaxlength();
+    const wrapperClasses = classnames('usa-form-group', {'usa-form-group--error': error });
+    const labelClasses = classnames('usa-label', {'usa-label--error': error });
 
     return (
-      <Host>
-        <label htmlFor="textarea">
+      <div class={wrapperClasses}>
+        <label htmlFor="textarea" class={labelClasses}>
           {label}
-          {required && <span class="required">{i18next.t('required')}</span>}
+          {required && <span class="usa-label--required">{i18next.t('required')}</span>}
         </label>
         {error && (
-          <span id="error-message" role="alert">
-            <span class="sr-only">{i18next.t('error')}</span> {error}
+          <span id="error-message" class="usa-error-message" role="alert">
+            <span class="usa-sr-only">{i18next.t('error')}</span> {error}
           </span>
         )}
         <textarea
+          class="usa-textarea"
```

Comment:
The `usa-textarea` package [doesn't have error state styling](https://github.com/uswds/uswds/blob/6dfc4689fda25ebeb7a6c6dc1f68a50c810c0d29/packages/usa-textarea/src/styles/_usa-textarea.scss#L3-L10). The [`usa-input` package does](https://github.com/uswds/uswds/blob/6dfc4689fda25ebeb7a6c6dc1f68a50c810c0d29/packages/usa-input/src/styles/_usa-input.scss#L8-L12), but the height of that is off. I feel like it would be strange to mix classes like `usa-textarea usa-input--error`, and in order to do that we would have to import CSS that will go unused.

Reply:
This is an interesting finding, I'm glad you made note of it. This could definitely affect our adoption of USWDS CSS for our components, especially if we need to go back and forth with them on requested changes.

Reply:
As a worst-case scenario, we could use what we can from USWDS and add whatever CSS we needed to our `va-textarea.scss` file.